src/StripeTests/Entities/StripeApplicationFeeTest.cs

```diff
@@ -0,0 +1,49 @@
+namespace StripeTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+
+    using Newtonsoft.Json;
+    using Stripe;
+    using Xunit;
+
+    public class StripeApplicationFeeTest : BaseStripeTest
+    {
+        [Fact]
+        public void Deserialize()
+        {
+            string json = GetFixture("/v1/application_fees/fee_123");
+            var applicationFee = Mapper<StripeApplicationFee>.MapFromJson(json);
+            Assert.NotNull(applicationFee);
+            Assert.IsType<StripeApplicationFee>(applicationFee);
+            Assert.NotNull(applicationFee.Id);
+            Assert.Equal("application_fee", applicationFee.Object);
+        }
+
+        [Fact]
+        public void DeserializeWithExpansions()
+        {
+            string[] expansions = {
+              "account",
+              "application",
+              "balance_transaction",
+              "charge",
+              "originating_transaction",
+            };
+
+            string json = GetFixture("/v1/application_fees/fee_123", expansions);
+            var applicationFee = Mapper<StripeApplicationFee>.MapFromJson(json);
+            Assert.NotNull(applicationFee);
+            Assert.IsType<StripeApplicationFee>(applicationFee);
+            Assert.NotNull(applicationFee.Id);
+            Assert.Equal("application_fee", applicationFee.Object);
+
+            Assert.Equal("account", applicationFee.Account.Object);
```

Comment:
You should probably add `Assert.NotNull(applicationFee.Account)` before this assert (and do something similar for all the following asserts), otherwise the test would crash if the attribute was null for some reason.

It's not a huge deal -- in both cases, the test would fail. It's just cleaner and more explicit to test with a failed assertion rather than with a null pointer exception :)